src/SelfAssessment/AssessmentQuestion.js

```diff
@@ -145,6 +142,33 @@ export const AssessmentQuestion = ({ onNext, onChange, option, question }) => {
   )
 }
 
+const AssessmentOptions = ({ option, question, selectedValues, onSelect }) => {
+  const displayAsOption = [
+    SCREEN_TYPE_CHECKBOX,
+    SCREEN_TYPE_RADIO,
+    SCREEN_TYPE_DATE,
+  ].includes(question.screen_type)
+
+  const mapAssessmentOptions = option.values.reduce((result, option, index) => {
+    if (option.label !== "None") {
+      result.push(
+        <AssessmentOption
+          answer={selectedValues.find((v) => v.index === index)}
+          index={index}
+          key={option.value}
+          onSelect={(value) => onSelect(value, index)}
+          option={option}
+          isSelected={selectedValues.some((v) => v.index === index)}
+          optionType={question.screen_type}
+        />,
+      )
+    }
+    return result
+  }, [])
+
+  return displayAsOption && mapAssessmentOptions
```

Comment:
How do you feel about returning an explicit null here? I feel like is a bit easier to read that we won't render the assessment option if it shouldn't be rendered as an option. 
Does it work if we short-circuit the method to return null as soon as we know that we shouldn't be rendering the options at all?

Reply:
Hmm... We can make this more explicit but I think there is no need. This is a known conditional rendering React pattern so I think it is safe to stick with it.